set firewall that will work in localized Windows

alantsai opened the pull request:
Set firewall using powershell with unique Name and Group.
Avoid problem in localized Windows

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @TerryLanfear, has been notified to review your proposed change.

TerryLanfear commented:
@4c74356b41 - Thank you for this update, your changes can be merged.

@BryanLa - This PR can be merged.

TerryLanfear commented:
@4c74356b41  - My apologies.  I am waiting for input from editorial.  Thank you.

@BryanLa - Please hold-off on merge.  #hold-off

TerryLanfear commented:
@4c74356b41 - Looks like this can go as you have it. Thank you for the update, your changes can be merged.

@BryanLa - This PR can be merged.

alantsai committed:
set firewall that will work in localized Windows

Set firewall using powershell using unique Name and Group,
avoid problem in localized Windows
